api/src/main/java/io/opencensus/trace/SpanBuilder.java

```diff
@@ -135,6 +137,23 @@
   public abstract SpanBuilder setRecordEvents(boolean recordEvents);
 
   /**
+   * If set to {@code true} this is equivalent with calling the {@link
+   * io.opencensus.trace.export.SampledSpanStore#registerSpanNamesForCollection(Collection)} in
+   * advance for the given span name.
+   *
+   * <p>WARNING: setting this option to a randomly generated span name can OOM your process because
+   * the library will save samples for each name.
+   *
+   * <p>It is strongly recommended to use the {@link
+   * io.opencensus.trace.export.SampledSpanStore#registerSpanNamesForCollection(Collection)} API
+   * instead.
+   *
+   * @return this.
+   */
+  @ExperimentalApi
+  public abstract SpanBuilder setRegisterNameToLocalSpanStore(boolean registerNameToLocalSpanStore);
```

Comment:
This method has nothing to do with "registration" per se. Something like `setSampleToLocalSpanStore()` is more accurate.

Reply:
done.